src/index.html

```diff
@@ -1,14 +1,268 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Human rights</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 class="title">
+      Universal Declaration of Human Rights
+    </h1>
+
+    <h2 class="subtitle">
+      Preamble
+    </h2>
+
+    <p class="parafraph">
+    Whereas recognition of the inherent dignity and of the equal and inalienable
+    rights of all members of the human family is the foundation of freedom, justice
+    and peace in the world,
+      <br>
+    Whereas disregard and contempt for human rights have resulted in barbarous
+    acts which have outraged the conscience of mankind, and the advent of a world
+    in which human beings shall enjoy freedom of speech and belief and freedom
+    from fear and want has been proclaimed as the highest aspiration of the common
+    people,
+      <br>
```

Comment:
All `p` and `li` tags in the document should also have correct indentations. Example:

```suggestion
    <p class="parafraph">
      Whereas recognition of the inherent dignity and of the equal and inalienable
      rights of all members of the human family is the foundation of freedom, justice
      and peace in the world,
      <br>
      Whereas disregard and contempt for human rights have resulted in barbarous
      acts which have outraged the conscience of mankind, and the advent of a world
      in which human beings shall enjoy freedom of speech and belief and freedom
      from fear and want has been proclaimed as the highest aspiration of the common
      people,
      <br>
```